### ISSUE TITLE
Label on HTML component doesn’t display

### ISSUE THREAD

**Issue by dwipper** [gradio-app/gradio#5861]
### Describe the bug

I’m using a HTML component to display html formatted information coming back from openai completions. The content displays fine, but there’s no Label being displayed for the field, which makes the styling in the UI look off. Is this the expected behavior of the component?

When I used a Textbox, the label displayed nicely.

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

Here’s my code for using the HTML component:
gr.Interface(fn=data_querying,
                 inputs=[gr.components.Textbox(lines=5, placeholder="Enter your question here", label="Question")],
                 outputs=[gr.components.HTML(label="Response",show_label=True,visible=True)
                     ],
                 description="Ask any question?",
                 allow_flagging="manual",
                 examples=ui_examples,
                 flagging_options=["The response is incorrect","The response is inappropriate","The response doesn't make sense"]
                 )

### System Info

```shell
Gradio Environment Information:
------------------------------
Operating System: Darwin
gradio version: 3.47.1
gradio_client version: 0.6.0

------------------------------------------------
gradio dependencies in your environment:

aiofiles: 23.2.1
altair: 5.1.1
fastapi: 0.103.2
ffmpy: 0.3.1
gradio-client==0.6.0 is not installed.
httpx: 0.25.0
huggingface-hub: 0.16.4
importlib-resources: 6.1.0
jinja2: 3.1.2
markupsafe: 2.1.3
matplotlib: 3.7.2
numpy: 1.25.2
orjson: 3.9.7
packaging: 23.1
pandas: 2.0.3
pillow: 10.0.0
pydantic: 1.10.13
pydub: 0.25.1
python-multipart: 0.0.6
pyyaml: 6.0.1
requests: 2.31.0
semantic-version: 2.10.0
typing-extensions: 4.5.0
uvicorn: 0.23.2
websockets: 11.0.3
authlib; extra == 'oauth' is not installed.
itsdangerous; extra == 'oauth' is not installed.


gradio_client dependencies in your environment:

fsspec: 2023.9.2
httpx: 0.25.0
huggingface-hub: 0.16.4
packaging: 23.1
requests: 2.31.0
typing-extensions: 4.5.0
websockets: 11.0.3
```

### Severity

I can work around it

**Comment by abidlabs** [comment]
Hi @dwipper this is expected actually -- the `HTML` and `Markdown` components do not show a box with the label, they just render the HTML that is provided. We could potentially add the label box here, but I don't think this is something we'd be able to do in the near future. If you'd like to open a PR for this, we can take a look!

**Comment by dwipper** [comment]
Hi @abidlabs I appreciate your quick response! I've uploaded some screen shots of my app. I'm certainly open to suggestions on how to make the UI consistent for the output side of the gr.Interface component. I've tried putting a gr.Markdown above the output HTML component, but the styling is still a challenge. If there was a way to replicate the typical field label styling in a Markdown field, that could be a solution?

![image](https://github.com/gradio-app/gradio/assets/140648761/8bdb8950-e98d-4372-9600-51444d3e2e97)
![image](https://github.com/gradio-app/gradio/assets/140648761/4e9df690-3d17-4a71-bee3-7599ddb33a5e)
![image](https://github.com/gradio-app/gradio/assets/140648761/d25581d4-cce4-46f9-b2cc-2223f6b174bb)
